lib/api_docs.rb

```diff
@@ -27,7 +27,8 @@ def update_docs(packages)
       {
         name: name,
         pkg: get_pkg(package),
-        docs: get_docs(package)
+        docs: get_docs(package),
+        title: get_docs(package)[0].longname,
```

Comment:
Since this method causes a read & parse, lets only do it once and store it in a local var for reference